membership/models.py

```diff
@@ -808,6 +814,14 @@ def save(self, *args, **kwargs):
     def is_reminder(self):
         return self.reminder_count > 0
 
+    def is_cancelled(self):
+        try:
+            if self.cancelledbill is not None:
+                return True
+        except CancelledBill.DoesNotExist:
+            pass
+        return False
```

Comment:
Would be better to either set a variable in try and except and return it OR return in both try and except. No big deal.